cara/apps/calculator/__init__.py

```diff
@@ -129,6 +129,40 @@ async def post(self):
         self.finish(report)
 
 
+class ConcentrationModelJsonResponse(BaseRequestHandler):
+    def check_xsrf_cookie(self):
```

Comment:
Normally, for such an important implementation, I would expect a comment here to tell the reader why we disable XSRF cookies. (i.e. the fact that this is an API not a form handler, etc.). Nothing too laborious, but enough to allow readers to have the full context.

Also, I assume the test would fail if this method got removed (which is highly possible if people think that it isn't doing anything)

Reply:
Good idea with the comment explaining why we disable XSRF cookies. I added a comment.